src/uu/df/src/table.rs

```diff
@@ -771,32 +744,14 @@ mod tests {
     #[test]
     fn test_row_formatter_with_round_up_usage() {
         let options = Options {
-            block_size: BlockSize::Bytes(1),
+            columns: vec![Column::Pcent],
```

Comment:
why did this change?

Reply:
The test is about verifying whether the values in the `Use%` column are round up, so only the `Use%` column is of interest. And the block size is removed because it is irrelevant for what's in the `Use%` column.